packages/cli/lib/ui.js

```diff
@@ -12,6 +13,19 @@ const uiLine = () => {
   logger.log('-'.repeat(50));
 };
 
+/**
+ * Returns an object that aggregates what the terminal supports (eg. hyperlinks and color)
+ *
+ * @returns {object}
+ */
+
+const getTerminalUISupport = () => {
+  return {
+    hyperlinks: supportsHyperlinks.stdout,
+    color: supportsColor.stdout.level >= 1,
```

Comment:
I'm not quite sure what check to perform here. 

`supportsColor.stout` returns the following object: 

```
{ level: 3, hasBasic: true, has256: true, has16m: true }
```

Should I simply check that the level is greater than or equal to 1 (which means the terminal has basic color)? Or would it be better to check whether the `hasBasic` boolean returns true? That seems to be enough for our purposes, since we're basically just using chalk to color text, correct? I wanted to check, though, in case I'm missing something.

Reply:
That makes sense to me. `hasBasic` sounds like a reasonable thing to check.